test/connectors/ManageDocumentsConnectorSpec.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import base.SpecBase
+import com.github.tomakehurst.wiremock.client.WireMock.aResponse
+import com.github.tomakehurst.wiremock.client.WireMock.post
+import com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo
+import generators.ModelGenerators
+import models.response.ResponseMovementMessage
+import org.scalacheck.Arbitrary.arbitrary
+import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
+import play.api.http.Status._
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.http.HttpResponse
+
+import scala.concurrent.Future
+
+class ManageDocumentsConnectorSpec extends SpecBase with WiremockSuite with ScalaCheckPropertyChecks with ModelGenerators {
+
+  override protected def portConfigKey: String = "microservice.services.manage-documents.port"
+
+  lazy val connector: ManageDocumentsConnector = app.injector.instanceOf[ManageDocumentsConnector]
+
+  implicit val headerCarrier: HeaderCarrier = HeaderCarrier()
+
+  "ManageDocumentsConnector" - {
+
+    "getUnloadingPermissionPdf" - {
+
+      "must return status Ok and PDF" in {
```

Comment:
Do we use the default exception throwing behaviour of http-verbs? If so can we override it here alone for now (there is a separate tech-debt ticket to fix the others)